apps/sertra/components/mod-selector/search.tsx

```diff
@@ -18,21 +18,15 @@ export const SearchBox = <T,>({
 }: SearchBoxProps<T>) => {
   const [isFocused, setFocused] = useState<boolean>(false);
 
-  function focusFirstOption(target: EventTarget & HTMLInputElement) {
-    if (
-      target.parentElement?.parentElement?.nextElementSibling?.children[0]
-        ?.children[0]
-    ) {
-      target.parentElement.parentElement.nextElementSibling.children[0].children[0].focus();
+  function focusFirstOption() {
+    if (optionsRef.current?.children[0]) {
```

Comment:
While the fix for useRef is implemented, you've seemed to ignore the comment about optional chaining operator. I guess we don't need to use it here, but please comment if the "wontfix" was intentional.